app/jinja_filters.py

```diff
@@ -82,7 +83,14 @@ def format_month_year_date(value, date_format='%B %Y'):
 
 @blueprint.app_template_filter()
 def format_datetime(value, date_format='%d %B %Y at %H:%M'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
+    date = datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f')
+    tz_time = pytz.timezone('UTC').localize(date)
+    london_tz = pytz.timezone('Europe/London')
+    london_time = tz_time.astimezone(london_tz)
+    time = london_time.strftime(date_format)
+    timezone = '(GMT+1)' if datetime.dst(london_time) == timedelta(hours=1) else '(GMT)'
+    return "<span class='date'>{date} {timezone}</span>".format(date=datetime.strptime(time, date_format).
+                                                                strftime(date_format), timezone=timezone)
```

Comment:
time is already in the string format you need so why are we converting it back to a date and then converting it back into a string. Just pass time in here.